lib/TPP/DefaultTppPasses.cpp

```diff
@@ -90,8 +90,10 @@ struct DefaultTppPasses
     if (linalgToVector) {
       skipOperations.push_back("all");
     }
-    if (vectorToXSMM)
-      skipOperations.clear();
+    if (vectorToXSMM) {
```

Comment:
Careful. `linalg-to-vector` and `vector-to-xsmm` could both be set, and this would result in a vector `{ "all", "transpose", "vnni" }`.

Reply:
Fixed.